MobiusCore/Source/BackwardsCompatibility.swift

```diff
@@ -39,6 +39,18 @@ public extension Next where Effect: Hashable {
     }
 }
 
+public extension Mobius.Builder {
+    @available(*, unavailable, message: "handle dispatching manually, or use MobiusController")
```

Comment:
“Manually” means there is no extension point to directly replace these things. Our previous recommendation was to never use `withEventQueue`/`withEffectQueue` since it was known that our scheduling didn’t work properly and needed to be replaced.

Reply:
OK, I think we should have a recommendation for how to handle this (in the controller, it's through the `eventFilter` I guess). But we can do that later on. So this is fine, IMO.